homeassistant/components/automation/config.py

```diff
@@ -79,8 +79,21 @@ async def async_validate_config_item(hass, config, full_config=None):
     return config
 
 
+class AutomationConfig(dict):
+    """Dummy class to allow adding attributes."""
```

Comment:
Should we do a dummy class or go for a class that can hold raw + validated? Do we expect to add more values in the future? Should this hold the traces??